pss/pss.go

```diff
@@ -29,6 +29,13 @@ type Pss struct {
 	localStore chunk.Store
 }
 
+// Monitor is used for tracking status changes in sent messages
+type Monitor struct {
```

Comment:
what does `states` represent? for each chunk state, what does its corresponding string mean? and why do we need this string?

Reply:
states are the internal tag states
The string is explicit to get the current state in a string format.
```
for state := range monitor.states {
		t.Log("message has been ", state)
	}
```

Reply:
this information is constant, we do not need to store a `map` inside the monitor to know that `stateStored`'s string representation is `stored`.

furthermore, this `map` is being created and filled with each new monitor (therefore repeated with each trojan chunk sent) and i don't see a reason for this at this point.

i suggest we do away with this field altogether